origin-discovery/src/apollo/listingMetadata.js

```diff
@@ -0,0 +1,38 @@
+// how frequently featured/hidden listings list updates
+const LISTINGS_STALE_TIME = 60 * 1000 //60 seconds
```

Comment:
nit: rename to METADATA_STALE_TIME or METADATA_STALE_THRESHOLD

Reply:
👍